content/devops/foundational-short-course/1-what-is-devops/_index.md

```diff
@@ -0,0 +1,19 @@
+---
+content_type: topic
+ready: true
+title: What is DevOps?
+---
+
+![](./devops-logo.png)
+
+DevOps is a methodology (processes, culture and tools) used to increase the level of communication and collaboration between **development teams** and **operation teams** to speed up the design, development, testing and delivery of high quality software aiming for frequent updates, uninterrupted availability, transparency and rapid feedback.
+
+It also expands to other teams within an organisation, like designers, product managers, quality assurance (QA), security and even end-users.
+
+DevOps builds on top of Agile (breaking large software development projects into smaller tasks, leading to faster build, test and feedback cycles) to consolidate historical siloed teams – developers and operations.
+
+### Other “xOps”: DevSecOps, DataOps.
```

Comment:
Should be a level 2 heading. 

Heading levels are not just about looks, it's also about accessability.

Reply:
Good catch, thanks.